plugins/lime-plugin-fbw/src/containers/scanPage/ScanList.js

```diff
@@ -20,22 +19,48 @@ export const ScanList = ({
 }) => {
 
 	const [expandedAps, setExpandedAps] = useState([]) 	// Expand scanned lists
+	const [getStatus, setGetStatus] = useState(false) 	// Activate get status
+	const [backendError, setBackendError] = useState(false) 	// Backend send false on perform an action
```

Comment:
In which real world case backendError will be true? Are we showing a useful message to the user in that situation?

In general, if a mutation should be considered to have been failed, then it should throw an error, so it's isError and error properties take the correct values.
Then you use those instead of local component state.

Reply:
Something like this seems to be better: 557e40f

Anyway, I guess i have to refactor backend side. Not sure if it has sense that if restart couldn't be performed return a `result: false` instead of throw an exception.